### DISCUSSION TITLE
florisboard 0.2.3 version   in playstore  not working

### DISCUSSION THREAD

**Discussion by HaiderClimaxCodeApp** [florisboard/florisboard#2239] [Q&A]
i am using florisboard 0.2.3 version  it working all other searching place  but only playstore first character type then not working

**Comment by patrickgold** [comment]
Version 0.2.3 is seriously outdated and not supported anymore. Use 0.3.16 for the latest stable build or the upcoming 0.4.0-alpha05 build for the latest alpha build.
